public/app/core/utils/userAgent.ts

```diff
@@ -0,0 +1,9 @@
+export function getBrowserOS(userAgent: string): string {
```

Comment:
There is a related issue #16222 to implement useragent parsing in the HTTP API that would be the proper way forward (I think). If you want to merge this PR as is before iterating further that's fine as a temporary thing.

Reply:
Thanks for pointing that issue out, Marcus. Will grab it as well.